src/locale.en.js

```diff
@@ -76,7 +77,7 @@ export const campsite = {
             secluded: 'Secluded',
             mountain_views: 'Mountain Views',
             winter_access: 'Accessible in Spring',
-            easy_access: 'Easy Access'
+            easy_access: 'Easy Access from Town'
```

Comment:
GH-97 -- I was finding that 'Easy Access' didn't really mean anything... is it easy access because it's an easy road? If so, then that's just anything that's easier than "High Clearance", no? Is it easy access because you can drive right up? Then that's anything that's not "Hike Access", right? What I finally landed on that could actually be a useful feature that's not mixed with what you can already learn from another attribute of the site is 'Easy Access from Town', which would indicate that you can get to/from town easily from this site. @amgoodman5 let me know if you can think of a better description for this tag.